### PR TITLE
Create Python package and use setup.py

### PR DESCRIPTION
**Pull request by ekeih**
Based on #1.

### CODE REVIEW

**Comment by ekeih** [comment]
Reintroduced the requirements.txt ;)

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-FROM python:3.6-alpine
+FROM python:3.7-alpine
```
Comment (markusressel) [inline Dockerfile]: Why not 3.8?

Comment (ekeih) [reply Dockerfile]: Fixed

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+setup(
+    name='frundenbot',
+    version=getenv('CIRCLE_TAG', default=datetime.now().strftime('%Y.%m.%d.dev%H%M%S')),
+    description='A Telegram Bot to watch your Freitagsrunde.',
+    long_description=readme(),
+    long_description_content_type='text/markdown',
+    url='https://github.com/ekeih/FrundenBot',
+    author='Max Rosin',
+    author_email='frundenbot@hackrid.de',
+    license='GPL',
```
[the remark this answers is not in the document]
Comment (ekeih) [reply setup.py]: I will replace it with the AGPLv3 in another MR.

**Pull request merged**